README.md

```diff
@@ -1,40 +1,92 @@
-Slick
-=====
+# Slick
 
 [![Maven](https://img.shields.io/maven-central/v/com.typesafe.slick/slick_2.13.svg)](http://mvnrepository.com/artifact/com.typesafe.slick/slick_2.13) [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/slick/slick?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 
-Slick is a modern database query and access library for Scala. It allows you
-to work with stored data almost as if you were using Scala collections while
-at the same time giving you full control over when a database access happens
-and which data is transferred. You can write your database queries in Scala
-instead of SQL, thus profiting from the static checking, compile-time safety
-and compositionality of Scala. Slick features an extensible query compiler
-which can generate code for different backends.
+Slick is a functional database library for Scala.
 
-The following database systems are directly supported for type-safe queries.
-These are the databases and driver versions that have explicit automated tests.
+It allows you to work with relational databases almost as if you were using 
+Scala collections, while at the same time giving you full control over when a 
+database access happens and what data is transferred. By writing your queries 
+in Scala you can benefit from the static type checking, compile-time safety, and 
+compositionality of Scala, while retaining the ability to drop down to raw SQL 
+where needed for custom or advanced database features.
+
+Slick also features an advanced query compiler which can generate SQL for a variety
+of different database engines from the same Scala code, allowing you to focus on
+application logic without worrying about database-specific syntax and quirks.
+
+## Resources
+
+- full documentation, including Scaladocs and more complex examples, can be 
```

Comment:
Curious if this is intentionally lowercase because you prefer it that way

Reply:
If you mean, should the first word of a bullet-point be lowercase or uppercase - I always tend to lowercase out of habit as much as anything, but arguably since these are standalone sentences they should be uppercased. Will fix.